lib/inventory/controller.go

```diff
@@ -28,6 +28,7 @@ import (
 
 	"github.com/gravitational/trace"
 	"github.com/jonboulle/clockwork"
+	"golang.org/x/time/rate"
```

Comment:
Should we use our own rate limiter (from lib/limiter)? 

I'm fine with the answer being no - I would have preferred we didn't write our own limiter to begin with.

Reply:
My general rule has been to use `lib/limiter` if you need its fancier features (e.g. source-ip tracking), but to stick to `x/time/rate` for basic rate limiting.  I'm not married to that philosophy, but like you I don't see the point in using the homegrown option when the standard option does everything I need.

Reply:
Sounds good, let's keep this as is.